hadoop-project/pom.xml

```diff
@@ -1303,7 +1306,20 @@
       <dependency>
         <groupId>org.mockito</groupId>
         <artifactId>mockito-core</artifactId>
-        <version>2.28.2</version>
+        <version>${mockito.version}</version>
```

Comment:
does it matter that mockito-all is a completely different version from the old or new version of mockito core?

Reply:
No it doesn't matter.